usecases/traverser/hybrid/hybrid_fusion.go

```diff
@@ -28,7 +28,7 @@ func FusionRanked(weights []float64, resultSets [][]*search.Result, setNames []s
 			}
 			tempResult := res
 			docId := tempResult.ID
-			score := weights[resultSetIndex] / float64(i+60) // TODO replace 60 with a class configured variable in the schema
+			score := float32(weights[resultSetIndex] / float64(i+60)) // TODO replace 60 with a class configured variable in the schema
```

Comment:
**question (non-blocking):** is it intended to be `float32` not  `float64` ?

Reply:
yes, everything else is stored in float32, so this has to be float32 as well.  I should probably switch the weights as well, but this works for now